config/aws/credentials

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
They're just there as the SDK needs them. They Drupal VM GoAWS doesn't actually need to authenticate.